pjlib/src/pjlib-test/ioq_tcp.c

```diff
@@ -402,10 +402,6 @@ static int compliance_test_0(const pj_ioqueue_cfg *cfg)
                 return -52;
             }
             pending_op -= status;
-
-            if (pending_op == 0) {
-                status = 0;
-            }
```

Comment:
Also, this does not look like a fixing-warning?
`pending_op == 0` seems to mean that all operations have been completed successfully.

Reply:
The intention of the original code is to reset the status to zero when the loop `while (pending_loop)` completes.

But status will always be overwritten by `status = pj_ioqueue_poll()` below.